ch-05/src/Solution551.java

```diff
@@ -0,0 +1,23 @@
+import java.util.Scanner;
+
+public class Solution551 {

```

Comment:
The program crashes if `str1` and `str2` are equal (so, they contain the same text). Please fix this issue.